index.html

```diff
@@ -29,196 +29,199 @@
+          </h1>
+          <p class="presentation">
+            I'm a software developer and I can help you build a product , feature
+            or website. Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don't hesitate to
+            contact me.
+          </p>
+          <p class="connect">LET'S CONNECT</p>
+          <div class="icons">
+            <ul>
+              <li>
+                <a href=""><i class="fa fa-twitter"></i></a>
+              </li>
+              <li>
+                <a href=""><i class="fa fa-linkedin"></i></a>
+              </li>
+              <li>
+                <a href=""><i class="fa fa-medium"></i></a>
+              </li>
+              <li>
+                <a href=""><i class="fa fa-github"></i></a>
+              </li>
+              <li>
+                <a href=""><i class="fa fa-angellist"></i></a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </section>
     </header>
-    <section class="main">
-      <div class="content">
-        <h1>
-          I'm Marie Grace.<br />Glad to see <br />
-          you!
-        </h1>
-        <p class="presentation">
-          I'm a software developer and I can help you build a product , feature
-          or website. Look through some of my work and experience! If you like
-          what you see and have a project you need coded, don't hesitate to
-          contact me.
-        </p>
-        <p class="connect">LET'S CONNECT</p>
-        <div class="icons">
-          <ul>
-            <li>
-              <a href=""><i class="fa fa-twitter"></i></a>
-            </li>
-            <li>
-              <a href=""><i class="fa fa-linkedin"></i></a>
-            </li>
-            <li>
-              <a href=""><i class="fa fa-medium"></i></a>
-            </li>
-            <li>
-              <a href=""><i class="fa fa-github"></i></a>
-            </li>
-            <li>
-              <a href=""><i class="fa fa-angellist"></i></a>
-            </li>
+    <section class="body">
+      <section class="works" id="section1">
+        <img class="photo" id="photo1" src="images/tonic project.png" />
+        <div class="part1">
+          <h1 class="title">Tonic</h1>
+          <ul class="canopy">
+            <li class="can"><b>CANOPY</b></li>
+            <li class="dev"><b>Back End Dev</b></li>
+            <li class="dev"><b> 2015</b></li>
           </ul>
+          <p class="presentation">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <div class="languages">
+            <ul>
+              <li class="lang">HTML</li>
+              <li class="lang">CSS</li>
+              <li class="lang">JavaScript</li>
+            </ul>
+          </div>
+          <button class="button">See project</button>
         </div>
-      </div>
-    </section>
```

Comment:
-[ ] I suggest you use the `div` tag instead of the `section` tag, note that you should use the `section` tag only for a main container. Kindly remove it to follow HTML's good semantics, and best practices and to make your project looks more professional.

- [ ] I also suggest you stick to the design template as much as possible, kindy follows this [link](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0-1&t=WrLIPsAPrp3ccntP-0) to take a look at the Figma design.

>### Your design
>
><img width="1041" alt="Screenshot 2023-05-05 at 1 55 34 PM" src="https://user-images.githubusercontent.com/91237525/236534733-0d13c0b8-c83b-40d4-965a-aee70e606859.png">
>
>### The Figma design
>
><img width="482" alt="Screenshot 2023-05-05 at 2 06 44 PM" src="https://user-images.githubusercontent.com/91237525/236534818-2c8a381a-8bf5-492b-b6e8-e0f4122a0888.png">